create_release.sh

```diff
@@ -0,0 +1,6 @@
+semantic-release --major version
```

Comment:
not sure a major version should be the default option, I think that it should be minor by default (because anything beyond that might mean we should alert stakeholders etc).

Either way, this should be configurable.